.gitignore

```diff
@@ -8,3 +8,4 @@
 /tools/ecs/*
 !/tools/ecs/composer.json
 !/tools/ecs/ecs.php
+/.idea
```

Comment:
Please use a global .gitignore file (`git config --global core.excludesFile`) to avoid IDE-specific directories here.